### PR TITLE
Updated SMS sample to send bulk SMS

### PR DESCRIPTION
**Pull request by avinashbr123**
Updated SMS sample project to send bulk SMS
Updated documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,40 @@ var twilioConfig = client.CreateTwilioProviderConfiguration(
            toPhoneNumber: toPhoneNumber,
            body: "A test to validate the provider config")));
 ```
+## Send SMS
+
+### Send single SMS
+
+```C#
+var sendSmsResult = await _smsClient.SendSmsAsync(subscriptionId, new Kmd.Logic.Sms.Client.Models.SendSmsRequest
```
Comment (nikhilcr939) [inline README.md:148]: Provide bulk sms client  function.

File: README.md [hunk 2]
```diff
@@ -140,6 +140,40 @@ var twilioConfig = client.CreateTwilioProviderConfiguration(
...
+    ToPhoneNumber = "put some phone number here",
+    callbackUrl = "provide your callback Url",
+    userData = "Can be anything upto 500 characters. If provided, this custom data will be passed to the callback"
+});
+```
+
+### Send Bulk SMS
+
+```C#
+var sendBulkSmsResult = await _smsClient.SendSmsAsync(subscriptionId, new Kmd.Logic.Sms.Client.Models.SendSmsRequest
```
Comment (nikhilcr939) [inline README.md:161]: Provide bulk sms client function.

File: README.md [hunk 3]
```diff
@@ -140,6 +140,40 @@ var twilioConfig = client.CreateTwilioProviderConfiguration(
...
+{
+    ToPhoneNumber = [
+        {
+            number = "put some phone number here"
```
Comment (nikhilcr939) [inline README.md:165]: Can we rewrite to more specific by saying - like first phone number, second phone number?

File: sample/Kmd.Logic.Sms.Client.Sample/CommandLineConfig.cs [hunk 4]
```diff
@@ -49,5 +51,17 @@ public class CommandLineConfig
...
         public string LinkMobilityCgiSource { get; set; } = "SMSSample";
+
+        public string UserData { get; set; } = "CustomerId=123";
+
+        // ToPhoneNumbers for sending bulk sms
+        public static List<ToPhoneNumber> ToPhoneNumbers()
+        {
+            return new List<ToPhoneNumber>
+            {
+            new ToPhoneNumber { Number = "Phone number" },
```
Comment (nikhilcr939) [inline sample/Kmd.Logic.Sms.Client.Sample/CommandLineConfig.cs:62]: Can we mention like "Phone Number - 1", "Phone Number - 2"?

File: sample/Kmd.Logic.Sms.Client.Sample/CommandLineAction.cs [hunk 5]
```diff
@@ -10,6 +10,7 @@ public enum CommandLineAction
         CreateFakeConfig,
         GetSms,
         SendSms,
+        SendBulkSms,
         SendSmsBatch,
```
Comment (avin101) [inline sample/Kmd.Logic.Sms.Client.Sample/CommandLineAction.cs:14]: Lets remove SendSmsBatch as we are providing Bulk SMS feature now.

**Review by avin101 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -140,6 +140,40 @@ var twilioConfig = client.CreateTwilioProviderConfiguration(
...
+    ToPhoneNumber = "put some phone number here",
+    callbackUrl = "provide your callback Url",
+    userData = "Can be anything upto 500 characters. If provided, this custom data will be passed to the callback"
+});
+```
+
+### Send Bulk SMS
+
+```C#
+var sendBulkSmsResult = await _smsClient.SendBulkSmsAsync(subscriptionId, new Kmd.Logic.Sms.Client.Models.SendSmsRequest
```
Comment (smahapatra86) [inline README.md:161]: Change the send request type to bulk one.

File: sample/Kmd.Logic.Sms.Client.Sample/Program.cs [hunk 7]
```diff
@@ -42,6 +42,7 @@ public static int Main(string[] args)
...
                         Log.Verbose("--Action=CreateLogicConfig --SubscriptionId={SubscriptionId} --LogicProviderSender={LogicProviderSender} ... --BearerToken={BearerToken}", "INSERT", "INSERT", "INSERT");
                         Log.Verbose("--Action=SendSms --ToPhoneNumber={ToPhone} --SubscriptionId={SubscriptionId} --ProviderConfigurationId={ProviderConfigurationId} ... --BearerToken={BearerToken}", "INSERT", "INSERT", "INSERT", "INSERT");
+                        Log.Verbose("--Action=SendBulkSms --SubscriptionId={SubscriptionId} --ProviderConfigurationId={ProviderConfigurationId} ... --BearerToken={BearerToken}", "INSERT", "INSERT", "INSERT", "INSERT", "...ToPhoneNumbers can be added in CommandLineConfig.cs");
                         Log.Verbose("--Action=SendSmsBatch --ToPhoneNumber={ToPhone} --NumberOfMessages={NumberOfMessages} --CallbackUri={CallbackUri} --SubscriptionId={SubscriptionId} --ProviderConfigurationId={ProviderConfigurationId} ... --BearerToken={BearerToken}", "INSERT", "INSERT", "INSERT", "INSERT", "INSERT", "INSERT");
```
Comment (smahapatra86) [inline sample/Kmd.Logic.Sms.Client.Sample/Program.cs:46]: We should remove this as well.

File: sample/Kmd.Logic.Sms.Client.Sample/Program.cs [hunk 8]
```diff
@@ -282,12 +240,29 @@ private static Guid SendSms(CommandLineConfig config)
...
+                subscriptionId: config.SubscriptionId,
+                request: new SendBulkSmsRequest(
+                    toPhoneNumbers: CommandLineConfig.ToPhoneNumbers(),
+                    body: config.SmsBody,
+                    callbackUrl: $"{config.CallbackUri}",
+                    providerConfigurationId: config.ProviderConfigurationId,
+                    userData: config.UserData));
+
+            Log.Information("Sent bulk SMS and got result {@Result}", sendSmsResult);
+            return (sendSmsResult as SendBulkSmsResponse)?.SmsMessageId ?? Guid.Empty;
```
Comment (smahapatra86) [inline sample/Kmd.Logic.Sms.Client.Sample/Program.cs:263]: Will it have single smsmessageid? It can have multiple as well.

**Review by smahapatra86 — APPROVED** [review]

**Pull request merged**